pkg/operator/object/ceph/rgw.go

```diff
@@ -343,13 +344,20 @@ func startDaemonset(context *clusterd.Context, store rookalpha.ObjectStore, vers
 }
 
 func rgwContainer(store rookalpha.ObjectStore, version string) v1.Container {
+	var hostName string
+	if store.Spec.Gateway.Host == "" {
+		// by default, the name of the service is used, with the namespace as a suffix
+		hostName = fmt.Sprintf("%s.%s", instanceName(store), store.Namespace)
```

Comment:
again, i would use the service name we have already created since that is resolvable.